src/include/firebird/FirebirdInterface.idl

```diff
@@ -734,6 +734,10 @@ version:	// 3.0.9 => 3.0.10, 4.0.1 => 4.0.2
 
 interface Provider : PluginBase
 {
+	const uint PTYPE_ANY		= 0;
```

Comment:
Method is `getType`, so why `PTYPE_` instead of `TYPE_`?
The names are already at scope `Provider`.
We don't use the extra specifications in the others constants.

Reply:
I will fix it - provided this method does not go away at all after "Redirection" cleanup.